src/shared/GlobalBanners/MissingDesignatedAdmins/MissingDesignatedAdmins.tsx

```diff
@@ -8,44 +8,18 @@ import { Provider } from 'shared/api/helpers'
 import A from 'ui/A'
 import Banner from 'ui/Banner'
 
-interface Props {
+interface MissingDesignatedAdminsProps {
   provider: Provider
-  hasAdmins?: boolean | null
-  isFetching?: boolean
-  isSelfHosted?: boolean
 }
 
-const useHideBanner = ({
+const MissingDesignatedAdmins: React.FC<MissingDesignatedAdminsProps> = ({
   provider,
-  hasAdmins,
-  isFetching,
-  isSelfHosted,
-}: Props) => {
-  if (!isSelfHosted || !provider || hasAdmins || isFetching) {
-    return true
-  }
-
-  return false
-}
-
-interface URLParams {
-  provider: Provider
-}
-
-const MissingDesignatedAdmins = () => {
-  const { provider } = useParams<URLParams>()
+}) => {
   const { data: hasAdmins, isFetching } = useSuspenseQueryV5(
     SelfHostedHasAdminsQueryOpts({ provider })
   )
-  // This hook is purely side stepping the complexity rule here.
-  const hideBanner = useHideBanner({
-    provider,
-    hasAdmins,
-    isFetching,
-    isSelfHosted: !!config.IS_SELF_HOSTED,
-  })
 
-  if (hideBanner) {
+  if (!config.IS_SELF_HOSTED || !provider || hasAdmins || isFetching) {
```

Comment:
we no longer need this !provider check either right?

Reply:
Nice catch, yea we can get rid of it, just a left over from a copy paste 🫣